onnxruntime/test/python/onnxruntime_test_float8.py

```diff
@@ -510,6 +517,58 @@ def test_model_qdq_cpu(self, name: str, float_name: str, saturate: int):
         self.assertEqual(expect.shape, y.shape)
         self.assertEqual(expect.dtype, y.dtype)
 
+    @unittest.skipIf(not hasattr(TensorProto, "FLOAT8E4M3FN"), reason="needs onnx>=1.14.0")
```

Comment:
I think it's possible to swap the order of skipIf and parameterized so the skip works correctly. Not sure if the current skip works because parameterized will replace the method with ones it creates? Never mind me if it does

Reply:
done